From e929665603966de386aa768af913e0fa58a68cc4 Mon Sep 17 00:00:00 2001
From: Stanislav Vlasic <svlasic@gmail.com>
Date: Tue, 27 Sep 2016 00:37:53 +0200
Subject: [PATCH] Modify makefile

---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 0647e05..d7d598e 100644
--- a/Makefile
+++ b/Makefile
@@ -383,6 +383,16 @@ KBUILD_CFLAGS   := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
 		   -Werror-implicit-function-declaration \
 		   -Wno-format-security \
 		   -fno-delete-null-pointer-checks \
+		   -Wno-error=maybe-uninitialized \
+		   -Wno-error=incompatible-pointer-types \
+          -Wno-maybe-uninitialized \
+          -Wno-incompatible-pointer-types \
+          -Wno-error=sizeof-array-argument \
+          -Wno-error=switch \
+          -Wno-error=aggressive-loop-optimizations \
+          -Wno-error=declaration-after-statement \
+          -Wno-error=frame-larger-than= \
+          -Wno-error=unused-function \
 		   -Werror \
 		   -std=gnu89
 
-- 
1.9.1

